### DISCUSSION TITLE
Indicator component issue

### DISCUSSION THREAD

**Discussion by UsamaEjaz0** [icflorescu/mantine-datatable#308] [Q&A]
The Indicator component is coming over the sticky table header. Using Mantine v5

![image](https://github.com/icflorescu/mantine-datatable/assets/45048065/08be3eb6-21ca-4076-bcb4-c83e5dc6b403)

**Comment by icflorescu** [comment]
The latest version of this package only supports Mantine 6.x, but that's not the issue.
Most likely the Indicator's `zIndex` CSS property is higher than the table header's one.
It would be helpful if you could provide a codesandbox (or another similar online) example.

**Comment by UsamaEjaz0** [comment]
Yes, I was able to fix the issue by providing a zIndex value of 1 to the Indicator.
